project/profiles/capacity4more/themes/c4m/kapablo/templates/page.tpl.php

```diff
@@ -128,6 +134,12 @@
           <?php if (!empty($breadcrumb)): print $breadcrumb; endif; ?>
         </div>
       </div>
+
+      <div class="row">
+        <div class="col-sm-12 col-md-12">
+          <?php print $image_banner; ?>
```

Comment:
@nirgn975 
on pages where the `og_context()` is `FALSE` we'll get an error over here.
We should check if the `$image_banner` is set...